src/App.tsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export const App: React.FC = () => {
+  const [keyPressed, setKeyPressed] = useState('');
+
+  const handleKeyDown = (e: KeyboardEvent) => {
+    setKeyPressed(e.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keyup', handleKeyDown);
+
+    return () => document.removeEventListener('keyup', handleKeyDown);
+  }, []);
+
+  return (
+    <div className="App">
+      {/* Keep this input for tests */}
+      <input type="text" className="hidden" />
+      {keyPressed
+        ? (
+          <p className="App__message">
+            The last pressed key is [
+            {keyPressed}
+            ]
+          </p>
+        )
+        : <p className="App__message">Nothing was pressed yet</p>}
```

Comment:
let's do it inside one `<p>`, I see no reason to conditionally render different tags here

Reply:
Done.
Thank you!